source/basic.tex

```diff
@@ -2233,8 +2233,8 @@
 
 \pnum
 \indextext{program}%
-A \defn{program} consists of one or more \defn{translation
-units} (Clause~\ref{lex}) linked together. A translation unit consists
+A \defn{program} consists of one or more \defn{translation units}
```

Comment:
I thought about changing this, but the term is a \defn today, and I don't think we have precedent for moving the 's' outside the \defn.  I certainly did not want to rewrite core wording so that this term might be re-rendered as singular, so opted for the lesser among evils,  leaving the term as defined.